.travis.yml

```diff
@@ -1,19 +1,22 @@
 language: node_js
+dist: bionic
```

Comment:
As long as you're upgrading, why not `focal`?

Reply:
Well, why not, did not even think about it while just trying to find working configuration :) 
I'll try it out.

Reply:
Did not get it working with focal, so I'm going to leave it as bionic.